presto-main/src/main/java/com/facebook/presto/operator/scalar/ScalarFunctionImplementation.java

```diff
@@ -54,13 +55,13 @@ public ScalarFunctionImplementation(
             Optional<MethodHandle> instanceFactory,
             boolean deterministic)
     {
-         this (ImmutableList.of(new ScalarImplementationChoice(nullable, argumentProperties, methodHandle, instanceFactory)), deterministic);
+        this (ImmutableList.of(new ScalarImplementationChoice(nullable, argumentProperties, methodHandle, instanceFactory)), deterministic);
     }
 
     /**
      * @param choices - must be ordered from generic to specific.
      * First choice is the default choice, which is the choice that is returned when legacy access method is used.
-     * The default choice must be usable for any invocation.
+     * The default choice must be usable for any arguments.
      * @param deterministic
      */
```

Comment:
There are specific conventions on javadoc. http://blog.joda.org/2012/11/javadoc-coding-standards.html

I rewrote it

```
    /**
     * Creates a ScalarFunctionImplementation consisting of one or more choices.
     * <p>
     * All choices must have the same SQL signature, and are equivalent in what they do.
     * The first choice is the default choice, which is the one used for legacy access methods.
     * The default choice must be usable under any context. (e.g. it must not use BLOCK_POSITION convention.)
     *
     * @param choices  the list of choices, ordered from generic to specific
     */